Makefile

```diff
@@ -25,7 +25,7 @@ sanity:
 	./hack/build-manifests.sh
 	git difftool -y --trust-exit-code --extcmd=./hack/diff-csv.sh
 
-build: build-operator build-csv-merger build-webhook
+build: build-operator build-csv-merger build-webhook build-functest
```

Comment:
Please remove `build-webhook`. Not in use here.

Reply:
Since we call `make build` in sanity checks as well, I reverted this part and call the targets in Dockerfile selectively.